scripts/add_stol_repos.sh

```diff
@@ -60,7 +60,7 @@ elif [[ ${GITHUB_REF_NAME} =~ release/.* ]]; then
 
 # check for release tags, do not use develop for these
 elif [[ $GITHUB_REF_TYPE = tag ]]; then
-    APT_CODENAME=release-${NO_UNDERSCORE_NAME##*/}
+    APT_CODENAME=main
```

Comment:
Looks like APT_CODENAME here will only support develop and main. Some packages were recently updated to look for a candidate version https://github.com/usdot-fhwa-stol/v2x-ros-conversion/blob/fc5f9aed3c8f74b0141c3c15b8578a043c0acafd/docker/install_dependencies.sh#L40 
so this should be updated to create it

Reply:
Please see the full file. Release candidates are also included